Cargo.toml

```diff
@@ -7,10 +7,10 @@ members = [
 resolver = "2"
 
 [workspace.dependencies]
-convert_case = "0.6.0"
+convert_case = "*"
 ocaml = "0.22.4"
-const-random = "0.1.13"
-paste = "1.0.9"
-proc-macro2 = "1.0.43"
-quote = "1.0.21"
-syn = { version = "1.0.99", features = ["full"] }
+const-random = "*"
+paste = "*"
+proc-macro2 = "*"
+quote = "*"
+syn = { version = "*", features = ["full"] }
```

Comment:
Note for the future: use
```
syn = { version = "2", features = ["full"] }
```
for the breaking change in the API (`Add` -> `Plus`).
semver _seems_ to be followed.

Reply:
See https://github.com/o1-labs/ocaml-gen/pull/39/commits/ac7724ebe5a404fe1d47f53cbc9b5697bc8ad866